sale_order_move_menu/views/sale_order_views.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="action_stock_move_move_menu" model="ir.actions.act_window">
+        <field name="name">Sale Moves</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">stock.move</field>
+        <field name="view_mode">tree,form</field>
+        <field name="search_view_id" ref="stock_move_move_menu_search"/>
+        <field name="view_ids" eval="[(5, 0, 0),
+            (0, 0, {'view_mode': 'tree', 'view_id': ref('stock_move_move_menu_tree')}),
+            (0, 0, {'view_mode': 'form', 'view_id': ref('stock.view_move_form')})]"/>
+        <!-- Default access is forbidden, this action shouldn't be directly accessed -->
+        <field name="domain">[('state','=','fake')]</field>
+        <field name="context">{
+            "search_default_filter_date": 1,
+        }</field>
+    </record>
+
+    <record model="ir.actions.server" id="action_stock_move_move_menu_srv">
+        <field name="name">Sale Moves</field>
+        <field name="model_id" ref="model_stock_move"/>
+        <field name="state">code</field>
+        <field name="code">
+            action = model.action_sale_moves()
+        </field>
+    </record>
+
+    <menuitem 
+        id="menu_sale_order_sale_moves"
+        name="Sale Moves" 
+        action="action_stock_move_move_menu_srv"
+        parent="sale.product_menu_catalog"
+        sequence="3"
```

Comment:
This sequence may be changed. At least, this is shown for v13:

<img width="519" height="205" alt="imagen" src="https://github.com/user-attachments/assets/bbd67bc9-e485-4a7a-84b8-5d9c3ff0ea17" />

and, for v17, order has changed:

<img width="588" height="258" alt="imagen" src="https://github.com/user-attachments/assets/b7197b62-5491-407e-9867-e13d8dc6a2a9" />